cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -0,0 +1,174 @@
+{% if options.action %}
+{% set action = options.action if options.action != '/' %}
+{% endif %}
+<form class="form-l"
+      {{ 'method=' ~ options.form_method if options.form_method else 'method=post' }}
+      {{ 'action=' ~ action if action else '' }}>
+    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+
+    {{ _render_form_wrapper(inputs) }}
+
+    <div class="form-actions form-actions__left-on-med">
+        <input class="btn form-actions_item"
+               type="submit"
+               value="Apply Filters">
+        <input class="btn btn__warning btn__link form-actions_item js-form_clear"
+               type="button"
+               value="Clear filters"
+               data-js-hook="form-clear">
+    </div>
+
+</form>
+{% endmacro %}
+
+{# A helper macro for filterable_list_controls.render()
+   ========================================================================== #}
+{% macro _render_form_wrapper(inputs) -%}
+
+    {% for input in inputs recursive %}
+        <div class="form-l_col
+                    form-l_col-{{ input.cols if input.cols else '1' }}">
+            <div class="form-group">
+
+                {% if input.label %}
+                <label class="form-label-header"
+                       {{ 'for=filter_' ~ input.id if input.id else '' }}>
+                    {{ input.label }}
+                </label>
+                {% endif %}
+
+                {% if input.type == 'form' %}
+                    <div class="form-l">
+                        {{ loop(input.children) }}
+                    </div>
+                {% elif input.type == 'email'
+                   or input.type == 'number'
+                   or input.type == 'password'
+                   or input.type == 'tel'
+                   or input.type == 'text'
+                   or input.type == 'url' %}
+
+                    <input type="{{ input.type }}"
+                           value="{{ input.attrs.value if input.attrs.value else '' }}"
+                           placeholder="{{ input.attrs.placeholder if input.attrs.placeholder else '' }}"
+                           {{ 'id=filter_' ~ input.id if input.id else '' }}
+                           {{ 'name=filter_' ~ input.name if input.name else '' }}>
+                {% elif input.type == 'checkbox' %}
+                    {% for checkbox in input.attrs -%}
+                        <label class="form-group_item"
+                               {{ 'for=filter_' ~ checkbox.id if checkbox.id else '' }}>
+                            {{ _filter_checkbox(checkbox.label,
+                                                checkbox.id,
+                                                checkbox.name,
+                                                checkbox.value) | safe }}
+                        </label>
+                    {% endfor %}
+                {% elif input.type == 'select' %}
+                    <label class="form-group_item">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/59346a46c913e1a01205ebe28d866f4ea8494780